lib/AST/ASTImporter.cpp

```diff
@@ -1577,6 +1577,9 @@ Decl *ASTNodeImporter::VisitStaticAssertDecl(StaticAssertDecl *D) {
   DeclContext *DC = Importer.ImportContext(D->getDeclContext());
   if (!DC)
     return nullptr;
+  Decl *AlreadyImportedD = Importer.GetAlreadyImportedOrNull(D);
```

Comment:
We could use `GetAlreadyImportedOrNull()` in `Decl *ASTImporter::Import(Decl *FromD)` as well instead of explicitly searching in `ImportedDecls`. There may be other cases where we do search explicitly in `ImortedDecls`. Could you please find those cases and refactor them to use the `GetAlreadyImportedOrNull()` ?

Reply:
Yes I can do it (at some point later).

Reply:
I found no other places to use `GetAlreadyImportedOrNull()`, only this one case.